dvc/analytics.py

```diff
@@ -2,12 +2,12 @@
 import logging
 import os
 
-from .env import DVC_NO_ANALYTICS
+from .env import DVC_ANALYTICS_HOST, DVC_NO_ANALYTICS
 
 logger = logging.getLogger(__name__)
 
 
-def collect_and_send_report(args=None, return_code=None):
+def collect_and_send_report(args=None, return_code=None, detach: bool = True):
```

Comment:
Similar to how we have in updater:
https://github.com/iterative/dvc/blob/477f266dc561a35f433b24bc29e56de8f0dbfbee/dvc/updater.py#L86-L93

Reply:
I also propose to introduce `DVC_DAEMON_DISABLE` envvar to run `updater` and `analytics` in-process.

Reply:
`DVC_NO_DAEMON` ?